middlewares/users.middleware.js

```diff
@@ -203,22 +215,31 @@ function usersMiddleware(datastore, errorResponse, secret, crypto, CONFIG) {
 
     function createUser(req, res, next) {
         const password = hashPassword(req.body.password);
-
+        var access_date = new Date();
+		access_date.setDate(access_date.getDate() - 1); // set to yesterday
         const key = datastore.key([ENTITY_KEY, req.body.email]);
         const entity = {
             key: key,
             excludeFromIndexes: ['phone', 'password_hash'],
             data: {
                 bid: {},
                 wishlist: [],
-                access: {},
+                access: {
+                	customer_next_payment_date: access_date,
+                	vendor_next_payment_date: access_date,
+                	customer_payment_amount: 500,
+                	vendor_payment_amount: 500,
+                	vendor_additional_paid: false
```

Comment:
I don't think this is a boolean property. We can have multiple additional payments

Reply:
I think we are not doing multiple credit, its just a check for creating an additional listing. If you change it to number then we need to provide more functionality like checking credits, buying additional credit etc. Right now the only way they get new credit is by creating additional listings, if for some reason the payment went through but creating the listing failed, he will have an extra credit, and we have to [tell him he has an existing credit] or [skip payment automatically], in both case a boolean is enough